<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>GO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>기업명 / 기업코드</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_top10">
    <property name="geometry">
     <rect>
      <x>243</x>
      <y>106</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TOP 10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_keyword">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>225</width>
      <height>125</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton_1">
       <property name="text">
        <string>시가총액</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>오늘의 유망주 (거래량, PER)</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="text">
        <string>영업이익률</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_4">
       <property name="text">
        <string>환산주가</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_5">
       <property name="text">
        <string>외국인 한도 소진율</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="table_top10">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>311</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>go_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>radio_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>radio_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>radio_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>radio_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>radio_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>go_clicked()</slot>
  <slot>radio_clicked()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
